pkg/operator/encryption/secrets.go

```diff
@@ -0,0 +1,211 @@
+package encryption
+
+import (
+	"encoding/base64"
+	"sort"
+	"strconv"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	apiserverconfigv1 "k8s.io/apiserver/pkg/apis/config/v1"
+	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
+	"k8s.io/klog"
+
+	"github.com/openshift/cluster-kube-apiserver-operator/pkg/operator/operatorclient"
+)
+
+func findSecretForKey(key keyAndMode, secrets []*corev1.Secret, targetNamespace string) *corev1.Secret {
+	if key == (keyAndMode{}) {
```

Comment:
hrm, why the () ?

Reply:
we discussed this before. Some Golang parser issue. It does not understand that the `{}` belong to the type.